### ISSUE TITLE
using mermaid with laravel

### ISSUE THREAD

**Issue by barttt** [knsv/mermaid#185]
Not sure how to address this problem
I am trying to use mermaid with a laravel view page. It seems to be trying to work but I cannot get rid of an error that causes the rendering to fail. I used bower to install mermaid and use this div:
					<div class="mermaid">
					graph TD; 
					sq[Square shape]-->di{Diamond}; 
					di-->A; 
					C-->di; 
					A-->C; 
					B-->C; 
					C-->ro2(Rounded square shape); 
					</div>

The stack looks like this:

[105]</module.exports.cloneCssStyles()mermaid.js (line 32075)
[104]</</render(id="mermaidChart0", txt="graph TD; \n sq[Squar...(Rounded square shape);", cb=function(svgCode), container=div.mermaid)mermaid.js (line 31907)
[104]</</exports.render(id="mermaidChart0", text="graph TD; \n sq[Squar...(Rounded square shape);", cb=function(svgCode), containerElement=div.mermaid)mermaid.js (line 31959)
[103]</</init()mermaid.js (line 31635)
[103]</</global.mermaid.init(sequenceConfig=undefined, nodes=undefined)mermaid.js (line 31670)
[103]</</exports.contentLoaded()mermaid.js (line 31712)
[103]</</<()mermaid.js (line 31732)

And the error reads "rule is undefined" on the line in question.
Can anyone help me out with a solution?
Thanks

**Comment by knsv** [comment]
The problem comes when trying to close the css rules it seems. Try
disabling the css cloning and see if that helps. With the 0.5.x version of
mermaid that can be done.

ex:
mermaid.initialize({mermaid:{cloneCssStyles: false}});

**Comment by barttt** [comment]
THank you for the quick reply.
I just did as you suggested (hopefully this is correct):. 
[code]
		mermaid.initialize({mermaid:{cloneCssStyles: false}});
		mermaid.initialize({startOnLoad:true});
		
[/code]
The result is the text is displayed above the rendering area,
 and the rendering is as before:
black filled boxes, arrows missing, no text in the diagram
and the same error..

This is trying to work, but it seems that something is interfering with the process.

**Comment by barttt** [comment]
Oops did not mean to close this issue..

**Comment by barttt** [comment]
Hi Knut-
Thanks for the reply..
I tried your suggestion. But the results were the same.
It occurred to me that perhaps the blade engine in laravel was causing some grief, so I renamed the file from welcome.blade.php to welcome.php, but that is not helping either. I have attached the web page text in case it can shed any light on this..

Thanks for any help

----------------------------------------
Bart Thompson
Software Engineer
Alpha Research & Technology
----------------------------------------
